GPTPullRequestReview/task.json

```diff
@@ -41,6 +41,29 @@
       "defaultValue": "false",
       "required": false,
       "helpMarkDown": "Select this option to support self-signed certificate."
+    },
+    {
+      "name": "prompt_instructions",
+      "type": "string",
+      "label": "prompt instructions for auto the code review",
+      "defaultValue": "Act as a code reviewer of a Pull Request, providing feedback on the code changes below. You are provided with the Pull Request changes in a patch format.             Each patch entry has the commit message in the Subject line followed by the code changes (diffs) in a unidiff format.",
+      "required": true,
+      "helpMarkDown": "use to set instructions for your reviewer will be injected into your system prompt"
+    },
+    {
+      "name": "model",
```

Comment:
it will be better if we have a predefined list of models instead of text

Reply:
Done.